config/config.go

```diff
@@ -13,8 +13,23 @@ import (
 
 // Config represents the JSON format of the config file.
 type Config struct {
-	BotToken string            `json:"bot_token"`
-	RickList []model.Snowflake `json:"ricklist"`
+	BotToken      string            `json:"bot_token"`
+	RickList      []model.Snowflake `json:"ricklist"`
+	RedisHost     string            `json:"redis_host"`
```

Comment:
Since these are all prefixed by "Redis" the right thing to do would probably be have a subconfig, but I don't think this is important and think you shouldn't do this. I just wanted to note it for the record.